superset-frontend/src/components/CachedLabel/index.tsx

```diff
@@ -44,6 +44,7 @@ const CacheLabel: React.FC<CacheLabelProps> = ({
       <Label
         className={`${className}`}
         type={labelType}
+        role="button"
```

Comment:
Following up on my higher level comment about fixing in low-level components, it looks like in this particular context, `Label` is pointing to `superset-frontend/src/components/Label/index.tsx`.

Could we simply for `role="button"` when `onClick` is defined? This would tackle all labels at once and prevent having to sprinkle the role prop across the code base, and forgetting to specify it in the future.

Reply:
Agreed